src/Main.java

```diff
@@ -1,5 +1,76 @@
+import java.util.Random;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello, World!");
+    public static void main(String[] args)
+    {
+        System.out.println("Задание № 1 ");
+        int total = 0;
+        int[] spent = new int [5] ;
+        //Scanner scan = new Scanner(System.in);
+        Random random = new Random();
+        for (int i = 0; i < spent.length  ; i++)
+        {
+            spent[i] = random.nextInt(100, 100000);
+            //System.out.println("Выплата  № " + (i + 1));
+            // spent[i] = scan.nextInt();
+        }
+        for (int i : spent)
+        {
+            total = total + i;
+        }
+        System.out.println("Сумма трат за месяц составила " + total + " рублей.");
+        System.out.println("Задание № 2 ");
+        int min = spent[1], max = 0;
+        for (int i: spent)
+        {
+            if (min > i) {
+                min = i;
+            }
+            if (max < i){
+                max = i;
+            }
+        }
+        System.out.println("Максимальная сумма трат за неделю составила " + max + " рублей.");
+        System.out.println("Минимальная сумма трат за неделю составила " + min + " рублей.");
+        System.out.println("Задание № 3 ");
+        float avarege = (float) (total / 4);
```

Comment:
Понял все правильно, только вместо хардкода 4 лучше взять размер массива (тем более, что там 5 элементов, а не 4)
```suggestion
        float avarege = (float) (total / spent.length);
```